tests/subsys/bootloader/external_flash/mcuboot_prj.conf

```diff
@@ -0,0 +1,61 @@
+CONFIG_CONSOLE_HANDLER=y
```

Comment:
What do all the configs here do? Could we live with just the defaults for most of them, or are they relevant to the test?

Reply:
No they are needed for MCUBoot this is MCUBoot's prj conf.

Reply:
So does this overwrite another prj.conf? Is it possible to supplement the configuration instead? For example passing the configs directly? Having a bunch of configs unrelated to the test is both not very educational and not very maintainable IMO.

Reply:
Doesn't overwrite. I don't think you can supplement since you are changing what is the source prj.conf for that sub image.